.github/workflows/check-pr.yml

```diff
@@ -8,12 +8,17 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [18.x]
+        node-version: [18.0.0]
```

Comment:
Is that necessary, as right now we have the setup-node action?

Reply:
fair point, I am going to remove this